docs/en/relying-party-solution.rst

```diff
@@ -0,0 +1,308 @@
+  * - **5**
+    - **Verifier (FE)**
+    - **-**     
+    - Inserts the reference to the previously generated object into a QR Code that is displayed on the screen
+  * - **6**
+    - **Verifier (FE)**
+    - **Wallet**
+    - QR Code is displayed ready to be scanned
+  * - **7**
+    - **Wallet**
+    - **-**
+    - Scans the QR Code
+  * - **8**
+    - **Wallet**
+    - **-**
+    - Extracts the Request URI from the payload of the QR Code
+  * - **9-10**
+    - **Wallet**
+    - **Verifier (BE)**          
+    - Requests the content of the Authorization Request by invoking the Request URI
+  * - **11**
+    - **Wallet**
+    - **-**     
+    - Verify the signature of the Request Object (JWT)
+  * - **12**
+    - **Wallet**
+    - **-**     
+    - Processes the content of the object and proceeds with the authentication
+  * - **13**
+    - **Wallet**
+    - **Verifier (BE)**     
+    - Generate an Authorization Response containing the VP Token
+  * - **14**
+    - **Verifier (BE)**
+    - **-**     
+    - Verify Authorization Response signature
+  * - **15**
+    - **Verifier (BE)**
+    - **-**     
+    - Process the content of the object, performs checks for integrity and tampering, and saves the Response to the persistence layer
+  * - **16**
+    - **Verifier (FE)**
+    - **Verifier (BE)**     
+    - Request a Session Cookie through polling, which will be available downstream after the checks performed in the previous step
+  * - **17**
+    - **Verifier (BE)**
+    - **-**     
+    - Consume the VP Token
+  * - **18**
+    - **Verifier (BE)**
+    - **-**     
+    - Requests the invalidation of the current session as the transaction has been successfully completed
+  * - **19**
+    - **Verifier (FE)**
+    - **User**     
+    - The user is successfully authenticated on the Verifier's website
+
+
+AuthorizationRequest Details
+----------------------------
+The AuthorizationRequest object is required by the Relying Party FrontEnd in order to show the QR Code. 
+The User will scan the QR Code using the Wallet granting the attributes to the RP.
+
+.. code-block:: javascript
+
+  POST /oid4vp HTTP/1.1
+  HOST: <VERIFIER_RELYING_PARTY_HOST>
+
+  HTTP/1.1 201 OK
+
+
+Below a non-normative response example
+
+.. code-block:: javascript
+
+  {
+    "alg": "ES256",
+    "typ": "JWT",
+    "x5c": [ "MIICajCCAdOgAwIBAgIC...20a" ],
+    "kid": "e0bbf2f1-8c3a-4eab-a8ac-2e8f34db8a47",
+    "trust_chain":[
+      "MIICajCCAdOgAwIBAgIC...awz",
+      "MIICajCCAdOgAwIBAgIC...2w3",
+      "MIICajCCAdOgAwIBAgIC...sf2"
+    ]
+  }
+  .
+  { 
+    "transactionId": "e9673f90-a652-4c47-8039-f60096cabafb",
+    "state": "a1252f90-a652-4c47-8039-f60096cavzqw",
+    "nonce": "o1923f90-a652-4c47-8039-f60096cabcza",
+    "requestUri": "oid4vp-wallet://authorize?client_id=https://<VERIFIER_RELYING_PARTY_HOST>&request_uri=https://<VERIFIER_RELYING_PARTY_HOST>/oid4vp/a1252f90-a652-4c47-8039-f60096cavzqw",
```

Comment:
The specification suggests to use direct_post in conjunction with request_uri, instead of redirect_uri used in same device flow.
Additionally, all params that aren't in snake_case are now changed